src/python/gudhi/simplex_tree.pyx

```diff
@@ -812,16 +812,19 @@ cdef class SimplexTree:
 
     def __eq__(self, other:SimplexTree):
         """Test for structural equality
+
         :returns: True if the 2 simplex trees are equal, False otherwise.
         :rtype: bool
         """
         return dereference(self.get_ptr()) == dereference(other.get_ptr())
     
     def __getstate__(self):
         """Pickle the SimplexTree data structure as a Python Byte Array
-        :raises MemoryError: In the case the serialization allocates a too large block of memory.
+
         :returns: Serialized SimplexTree data structure
         :rtype: numpy.array of shape (n,)
+
+        :raises MemoryError: In the case the serialization allocates a too large block of memory.
```

Comment:
I don't think we documented it for other functions. It doesn't hurt, but it may not be that useful either.

Reply:
I removed the exception documentation on [a3e9c0d](https://github.com/GUDHI/gudhi-devel/pull/871/commits/a3e9c0ddfa35f70b90e68227b06ca2f4e4531f64)